src/schema/nmdc.yaml

```diff
@@ -324,13 +335,30 @@ classes:
       - websites
       - publications
       - type  # custom slot that specifies object type
+      # external IDs
+      - INSDC bioproject identifiers
+      - INSDC SRA ENA study identifiers
+      - GOLD study identifiers
+      - MGnify project identifiers
     slot_usage:
       doi:
         description: >-
           The dataset citation for this study
+    exact_mappings:
+      - SIO:001066  ## study
+      - NCIT:C63536 ## study
+      - OBI:0000066 ## investigation
```

Comment:
`OBI:0000066` is a kind of planned process.  
`NCIT:C63536` is a kind of research activity.
`SIO:001066` is a kind of process.

Our description for study doesn't seem to capture the process sense of 'study' expressed in these concepts:
```
A study summarizes the overall goal of a research initiative and outlines the key objective of its underlying projects.
```

Reply:
Thanks! I'm not so concerned with differences in upper ontology classification, only with whether they capture the same concept or not.

Reply:
Sorry, I wasn't meaning to sound like was going down the ontological rabbit hole. I was only pointing out that the things we are mapping too may not line up well with the sense of 'study' we are using in the NMDC.